packages/esm-patient-search-app/src/compact-patient-search-extension/index.tsx

```diff
@@ -21,48 +22,40 @@ const CompactPatientSearchComponent: React.FC<CompactPatientSearchProps> = ({
   buttonProps,
 }) => {
   const { t } = useTranslation();
+  const config = useConfig<PatientSearchConfig>();
+  const inputRef = useRef<HTMLInputElement>(null);
+  const bannerContainerRef = useRef(null);
   const [searchTerm, setSearchTerm] = useState(initialSearchTerm);
-  const handleChange = useCallback((val) => setSearchTerm(val), [setSearchTerm]);
   const showSearchResults = useMemo(() => !!searchTerm?.trim(), [searchTerm]);
-  const config = useConfig();
   const patientSearchResponse = useInfinitePatientSearch(searchTerm, config.includeDead, showSearchResults);
   const { data: patients } = patientSearchResponse;
 
-  const handleSubmit = useCallback((evt) => {
-    evt.preventDefault();
-  }, []);
-
-  const handleClear = useCallback(() => {
-    setSearchTerm('');
-  }, [setSearchTerm]);
+  const handleChange = useCallback((val) => setSearchTerm(val), [setSearchTerm]);
 
-  const handleReset = useCallback(() => {
-    setSearchTerm('');
-  }, [setSearchTerm]);
+  const handleClear = useCallback(() => setSearchTerm(''), [setSearchTerm]);
 
-  // handlePatientSelection: Manually handles everything that needs to happen when a patient
-  // from the result list is selected. This is used for the arrow navigation, but is not used
-  // for click handling.
+  /**
+   * handlePatientSelection: Manually handles everything that needs to happen when a patient
+   * from the result list is selected. This is used for the arrow navigation, but is not used
+   * for click handling.
+   */
   const handlePatientSelection = useCallback(
-    (evt, index: number) => {
-      evt.preventDefault();
+    (event, index: number) => {
+      event.preventDefault();
       if (selectPatientAction) {
         selectPatientAction(patients[index].uuid);
       } else {
         navigate({
-          to: `${interpolateString(config.search.patientResultUrl, {
```

Comment:
`interpolateString` returns a string, so there's no need for the additional interpolation.